draft-ietf-mls-protocol.md

```diff
@@ -1059,8 +1059,9 @@ These effects are described in their respective subsections of {{proposals}}.
 The following general rules apply:
 
 * The `group_id` field is constant
-* The `epoch` field increments by one for each Commit message that
-  is processed
+* The `epoch` field is set to the first four octets of the hash of  the
```

Comment:
- Stop truncating hashes!
- What about the first epoch?

Reply:
I am more concern about loosing lexicographic ordering here. The DS wouldn't know how to reorder.